homeassistant/components/sense/sensor.py

```diff
@@ -235,24 +260,29 @@ def __init__(
         data,
         name,
         sensor_type,
-        is_production,
+        variant_id,
+        variant_name,
         trends_coordinator,
         unique_id,
     ):
         """Initialize the Sense sensor."""
-        name_type = PRODUCTION_NAME if is_production else CONSUMPTION_NAME
-        self._attr_name = f"{name} {name_type}"
+        self._attr_name = f"{name} {variant_name}"
         self._attr_unique_id = unique_id
         self._data = data
         self._sensor_type = sensor_type
         self._coordinator = trends_coordinator
-        self._is_production = is_production
+        self._variant_id = variant_id
         self._had_any_update = False
 
+        if variant_id in [PRODUCTION_PCT_ID, SOLAR_POWERED_ID]:
+            self._attr_native_unit_of_measurement = PERCENTAGE
```

Comment:
What % does this represent? 

I think that we should consider not adding all these entities by default. Please disable by deafult a couple of non-relevant ones by setting `self._attr_entity_registry_enabled_default = False`

Reply:
I have[ a PR open on the docs repo](https://github.com/home-assistant/home-assistant.io/pull/18983) that explains the editions.

In this case, these two sensors are reported as percentage values from the API for...
*  Net production as a percentage value ( energy produced / energy consumed * 100 )
* Solar powered percentage values, which is supposed to be a representation of the amount of power produced from solar that went to power the household and not exported to the grid ( 100 * (usage - from_grid) / usage )

From Sense community posts, the solar powered % number is easily the least interesting, and I will admit that it was just something I added because it was there. 

I'm working toward having `OptionsFlowHandlers` that would allow for switching on or off entire sensor classes, like having active sensors, but turning off per-device sensors, as well as letting users choose which trend sensors they want to be enabled.
 
That said, it's easier to type all that then it is to implement it, and I'm at the the `Hello World` stage of adding the flows themselves. It's going to take more time for me to get them fully implemented, figure out how to translate those choices into the actually reconfiguration of the integration, and then figure out how to update the tests.

Reply:
Let's disable these by default.

```suggestion
            self._attr_native_unit_of_measurement = PERCENTAGE
            self._attr_entity_registry_enabled_default = False
```

Reply:
Home Assistant already has options to disable entities, we shouldn't duplicate those features with an options flow.